agent/iskakalunan.py

```diff
@@ -66,7 +66,7 @@ def run(self):
             subprocess.call(['python', 'runtest.py', '-h', '>', 'result'])
             return
 
-        subprocess.call(['git', 'checkout', 'master'])
+        subprocess.call(['git', 'checkout', 'dev'])
```

Comment:
@Conjuror  if you want this parameter able to read from config, you can just simply add the key value pair in the config file, and you can get that variable from kwargs in __init__ function.